README.md

```diff
@@ -1,210 +1,182 @@
 
-## Examples
-
-```vim
-nnoremap <Leader>p <cmd>lua require'telescope.builtin'.git_files{}<CR>
-```
 
-Searches over files in a git folder. Note: This does not work outside a git repo folder.
+### Quick start 
+---
 
-```vim
-nnoremap <Leader>p <cmd>lua require'telescope.builtin'.find_files{}<CR>
-```
+To test if `telescope.nvim` is installed correctly try `:Telescope find_files<cr>`. 
 
-Search over files in your `cwd` current working directory.
+```viml
+" Find files using Telescope command-line sugar.
+nn <leader>ff <cmd>Telescope find_files<cr>
+nn <leader>fg <cmd>Telescope live_grep<cr>
+nn <leader>fb <cmd>Telescope buffers<cr>
+nn <leader>fh <cmd>Telescope help_tags<cr>
 
-```vim
-nnoremap <silent> gr <cmd>lua require'telescope.builtin'.lsp_references{}<CR>
+" Using lua functions
+nn <leader>ff <cmd>lua require('telescope.builtin').find_files()<cr>
+nn <leader>fg <cmd>lua require('telescope.builtin').live_grep()<cr>
+nn <leader>fb <cmd>lua require('telescope.builtin').buffers()<cr>
+nn <leader>ff <cmd>lua require('telescope.builtin').help_tags()<cr>
 ```
 
-Search over variable references from your Language Server.
+For a complete list of the builtin functions, see [builtin functions](#functions)
 
-```vim
-nnoremap <Leader>en <cmd>lua require'telescope.builtin'.find_files{ cwd = "~/.config/nvim/" }<CR>
-```
+## Configuration 
 
-Find all the files in your nvim config.
+### options 
 
-### Available keys for `defaults`
+Here a list of default options to play with in order to customize your
+experience and how telescope behaves globally (i.e. effecting builtin functions
+as well as custom ones). 
 
-- `generic_sorter`:
-    - Description: The sorter to be used for generic searches.
-    - `default`: `require('telescope.sorters').get_generic_fuzzy_sorter
-- `file_sorter`:
-    - Description: The sorter to be used for file based searches.
-    - `default`: `require('telescope.sorters').get_fuzzy_file
+| Keys                   | Options                    | Description                                           |
+|------------------------|----------------------------|-------------------------------------------------------|
+| `vimgrep_arguments`    | dict                       | The command line argument for grep search ... TODO.   |
+| `prompt_position`      | top/bottom                 | Where the prompt should be located.                   |
+| `prompt_prefix`        | string                     | The prefix prefix.                                    |
+| `sorting_strategy`     | descnding/ascending        | Where first selection should be located.              |
+| `layout_strategy`      | centr/horizontal/vertical  | How the telescope is drawn.                           |
+| `layout_defaults`      | TODO                       | Layout specific configuration ........ TODO           |
+| `selection_strategy`   | follow/reset/line          | ... TODO                                              | 
```

Comment:
Just because i'm currently working on it. Shouldn't this be `follow` / `reset` / `row`.
https://github.com/nvim-lua/telescope.nvim/blob/051aefdb8cea71294e6e62e8a4a1e051c48e4edd/lua/telescope/pickers.lua#L459-L474
Should be also wrong in current README

Reply:
@Conni2461 oh so the options are follow/reset/row, how did I miss that :dagger: